src/App.js

```diff
@@ -13,13 +13,18 @@ function App() {
       var data = await fetch(apiUrl).then((res) => {
         return res.json();
       });
-      setItems(data.list.item);
```

Comment:
data.list.item was the older configuration. That's how the data is built anymore